programs/cleos/main.cpp

```diff
@@ -2396,8 +2440,10 @@ int main( int argc, char** argv ) {
       shouldSend = false;
       set_code_callback();
       set_abi_callback();
-      std::cerr << localized("Publishing contract...") << std::endl;
-      send_actions(std::move(actions), 10000, packed_transaction::zlib);
+      if (actions.size()) {
+         std::cerr << localized("Publishing contract...") << std::endl;
+         send_actions(std::move(actions), 10000, packed_transaction::zlib);
+      }
```

Comment:
I think we should have a final message here if `actions.size()` is 0.  Sure, you should see two "Skipping..." messages but may as well also be explicit about the fact that no transaction is sent if everything is skipped

Reply:
sure will fix that.